doc/apimanager-reference.md

```diff
@@ -278,7 +278,7 @@ that should be set on it.
 | --- | --- | --- | --- | --- | --- |
 | Enabled | `enabled` | bool | No | `false` | Enable to use external system database, backend redis, system redis and apicast redis databases|
 
-When HighAvailability is enabled the following secrets have to been
+When HighAvailability is enabled the following secrets have been
```

Comment:
I think this should actually be:
`the following secrets have to be pre-created by the user:`

Reply:
Change incorporated. Thanks.